testing/src/main/java/org/oppia/android/testing/espresso/ImageViewMatcher.kt

```diff
@@ -0,0 +1,32 @@
+package org.oppia.android.testing.espresso
+
+import android.view.View
+import android.widget.ImageView
+import org.hamcrest.Description
+import org.hamcrest.TypeSafeMatcher
+
+/** A custom class that provides matching for ImageView. */
+class ImageViewMatcher {
```

Comment:
Created a specific matcher file for imageview.

Reply:
Ack.